core/templates/dev/head/editor/feedback_tab/feedback_tab.html

```diff
@@ -173,21 +175,28 @@
       <div class="oppia-suggestion-review-panel-container" style="float: left;">
         <strong>Current:</strong>
         <div class="oppia-suggestion-review-panel">
-          <p style="padding: 5px;"><[ oldContent ]></p>
+          <div angular-html-bind="oldContent"></div>
         </div>
       </div>
       <div class="oppia-suggestion-review-panel-container" style="float: right;">
         <strong>Suggested:</strong>
         <div class="oppia-suggestion-review-panel">
-          <p style="padding: 5px;"><[ newContent ]></p>
+          <div angular-html-bind="newContent"></div>
         </div>
       </div>
     </section>
+    <div style="margin-top: 20px; margin-left: 10px;" ng-show="canAccept">
+      Commit Message (required if accepting):
+      <input type="text" ng-model="commitMessage" style="width: 100%">
+    </div>
   </div>
 
   <div class="modal-footer">
+    <div ng-show="errorMessage" style="text-align: right; width: 100%; color: red; margin-bottom: 10px;">
+      <[ errorMessage ]>
+    </div>
     <button class="btn btn-default" ng-click="cancelReview()">Cancel</button>
-    <button class="btn btn-danger" ng-click="rejectSuggestion()" ng-disabled="!(activeThread.suggestion.status == 'new')">Reject</button>
-    <button class="btn btn-success" ng-click="acceptSuggestion()" ng-disabled="!(activeThread.suggestion.status == 'new')">Accept</button>
+    <button class="btn btn-danger" ng-click="rejectSuggestion()" ng-disabled="!canReject">Reject</button>
+    <button class="btn btn-success" ng-click="acceptSuggestion()" ng-disabled="!canAccept || commitMessage.length == 0">Accept</button>
```

Comment:
Sorry, I just saw this, must've missed it. Is this because the commit message is empty? I can make the commit message box red if it's blank to indicate something's wrong (also, commit message label will be changed to 'Brief Description of Changes').